v21.1/schema-design-database.md

```diff
@@ -0,0 +1,135 @@
+
+### Database best practices
+
+Here are some best practices to follow when creating and using databases:
+
+- Do not use the preloaded `defaultdb` database. Instead, create your own database with a `CREATE DATABASE` statement, and change it to the SQL session's [current database](sql-name-resolution.html#current-database) by executing a `USE [databasename];` statement, by passing the `--database=[databasename]` flag to the [`cockroach sql` command](cockroach-sql.html#general), or by specifying the `database` parameter in the [connection string](connection-parameters.html#connect-using-a-url) passed to your database schema migration tool.
+- Create databases as the `root` user, and create all other lower-level objects with a [different user](schema-design-overview.html#controlling-access-to-objects), with more limited privileges, following [authorization best practices](authorization.html#authorization-best-practices).
+- Limit the number of databases you create. If you need to create multiple tables with the same name in your cluster, do so in different [user-defined schemas](#create-a-user-defined-schema) in the same database.
+
+{% include {{page.version.version}}/sql/dev-schema-changes.md %}
+
+### Example
+
+Open a terminal window, and use the CockroachDB [SQL client](cockroach-sql.html) to open the [CockroachDB SQL shell](cockroach-sql.html#sql-shell) to your cluster:
+
+<section class="filter-content" markdown="1" data-scope="cockroachcloud">
+
+{% remote_include https://raw.githubusercontent.com/cockroachdb/docs/master/_includes/cockroachcloud/connect_to_crc.md|<!-- BEGIN CRC free sql -->|<!-- END CRC free sql --> %}
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="local">
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach sql \
+--certs-dir=[certs-directory]
+~~~
+
+</section>
+
+This command opens a SQL shell as the `root` user. Any statements executed from the shell will be executed by this user.
+
+To create a database as `root`, issue a `CREATE DATABASE` statement in the SQL shell:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE DATABASE cockroachlabs;
+~~~
+
+This statement creates a database object with the name `cockroachlabs`.
+
+To view the database in the cluster, use a [`SHOW DATABASES`](show-databases.html) statement:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SHOW DATABASES;
+~~~
+
+~~~
+  database_name
+-----------------
+  cockroachlabs
+  defaultdb
+  postgres
+  system
+(4 rows)
+~~~
+
+To follow [authorization best practices](authorization.html#authorization-best-practices), after you create a new database, you should also create a new SQL user to manage the lower-level objects in the new database.
+
+In the open SQL shell, execute the following `CREATE USER` statement:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE USER max;
```

Comment:
Minor nit here, but in some of the Hello World examples, the username is `maxroach`.